src/Microsoft.VisualStudio.Editors/OptionPages/GeneralOptions.vb

```diff
@@ -1,18 +1,28 @@
 ﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+Option Strict Off
```

Comment:
I had to add Option Strict Off , because it threw compile exception for this line Dim vsSetupConfig = New SetupConfiguration() , where we try to instantiate embedded object. It could not cast SetupConfiguration to it's embedded type.

---
In reply to: [192156136](https://github.com/dotnet/project-system/pull/3602#discussion_r192156136) [](ancestors = 192156136)

Reply:
Any other ways to use embedded objects in VB?

---
In reply to: [192230241](https://github.com/dotnet/project-system/pull/3602#discussion_r192230241) [](ancestors = 192230241,192156136)

Reply:
You shouldn't need Option Strict off for NoPIA - I suspect something else is going on, and we should work through that.